index.html

```diff
@@ -192,7 +192,7 @@ <h2 class="p-muted-heading u-align-text--center">Who&rsquo;s using Vanilla</h2>
   <div class="row">
     <div class="col-3">
       <h2 class="p-heading--three">Contribute</h2>
-      <p><a class="github-button" href="https://github.com/vanilla-framework" data-size="large" data-show-count="true" aria-label="Follow @vanilla-framework on GitHub">Follow @vanilla-framework</a></p>
+      <p><a class="github-button" href="https://github.com/canonical-web-and-design/vanilla-framework" data-size="large" data-show-count="true" aria-label="Follow @vanilla-framework on GitHub">Follow @vanilla-framework</a></p>
       <p><a class="github-button" href="https://github.com/vanilla-framework/vanilla-framework" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star vanilla-framework/vanilla-framework on GitHub">Star</a></p>
```

Comment:
While we are at it, we could probably update the star icon as well. Or just remove it? It seems to mostly duplicate what we already show above.

Reply:
Will just remove as it serves the same number as follow 😄